models/gc_spider_baseline/utils/SpiderBaselineRunner.py

```diff
@@ -0,0 +1,155 @@
+Author: Sil van de Leemput
+Email:  sil.vandeleemput@radboudumc.nl
+-------------------------------------------------------------
+"""
+from pathlib import Path
+import shutil
+import os
+import json
+
+import SimpleITK
+import numpy as np
+
+from mhubio.core import Instance, InstanceData, IO, Module
+
+# The GC Spider baseline algorithm is imported as a self-contained algorithm class with an execute method
+from TestSpine import Module as SpiderAlgorithm
+
+
+@IO.Config('traversal_direction_up', bool, True, the="Direction to traverse the image")
+@IO.ConfigInput('in_data', 'nifti|nrrd|mha:mod=ct|mr', the='supported datatypes for the spider baseline model')
+class SpiderBaselineRunner(Module):
+
+    traversal_direction_up: bool
+
+    SPIDER_DATA_DIR = Path(os.environ["VERSEG_BASEDIR"])
+    SPIDER_INTERNAL_DATASET_PATH = SPIDER_DATA_DIR / "datasets" / "spider_input"
+    SPIDER_EXPERIMENT_DIR = SPIDER_DATA_DIR / "experiments" / "SPIDER-Baseline"
+    SPIDER_INTERNAL_OUTPUT_DIR = SPIDER_EXPERIMENT_DIR / "results" / "spider_input"
+    SPIDER_EXPERIMENT_ARGUMENTS_FILE = SPIDER_EXPERIMENT_DIR / "arguments.json"
+    SPIDER_INTERNAL_DATASET_META_FILE = SPIDER_INTERNAL_DATASET_PATH / "metadata.json"
+
+    @IO.Instance()
+    @IO.Input('in_data', the='input sagittal spine image')
+    @IO.Output(
+        'out_data_raw',
+        'spider_baseline_vertebrae_segmentation_raw.mha',
+        'mha:mod=seg:model=SpiderBaselineSegmentation:seg=raw',
+        data='in_data',
+        bundle='model',
+        the='Spider baseline vertebrae segmentation for the input sagittal spine MRI. Raw segmentation output: '
+            '0: background '
+            '1-25: different vertebrae numbered from the bottom (i.e. L5 = 1) '
+            '100: spinal canal '
+            '101-125 for partially visible vertebrae '
+            '201-225: different intervertebral discs (also numbered from the bottom, i.e. L5/S1 = 201)) '
+    )
+    @IO.Output(
+        'out_data',
+        'spider_baseline_vertebrae_segmentation.mha',
+        'mha:mod=seg:model=SpiderBaselineSegmentation:seg=remapped:roi=VERTEBRAE_L5,VERTEBRAE_L4,VERTEBRAE_L3,VERTEBRAE_L2,VERTEBRAE_L1,VERTEBRAE_T12,VERTEBRAE_T11,VERTEBRAE_T10,VERTEBRAE_T9,VERTEBRAE_T8,VERTEBRAE_T7,VERTEBRAE_T6,VERTEBRAE_T5,VERTEBRAE_T4,VERTEBRAE_T3,VERTEBRAE_T2,VERTEBRAE_T1,VERTEBRAE_C7,VERTEBRAE_C6,VERTEBRAE_C5,VERTEBRAE_C4,VERTEBRAE_C3,VERTEBRAE_C2,VERTEBRAE_C1',
+        data='in_data',
+        bundle='model',
+        the='Spider baseline vertebrae segmentation for the input sagittal spine MRI. Remapped segmentation output: '
+            '0: background '
+            '1-25: different vertebrae numbered from the bottom (i.e. L5 = 1) (includes partially visible vertebrae from raw segmentation)'
+    )
+    def task(self, instance: Instance, in_data: InstanceData, out_data_raw: InstanceData, out_data: InstanceData) -> None:
+        in_data_path = Path(in_data.abspath)
```

Comment:
Does the `Path` wrapping happen for any reason? If no, please remove as it decreases the readablity.

Reply:
I'll remove it